ee/clickhouse/queries/funnels/base.py

```diff
@@ -119,9 +124,39 @@ def _format_single_funnel(self, results, with_breakdown=False):
             else:
                 serialized_result.update({"average_conversion_time": None, "median_conversion_time": None})
 
+            # Construct converted and dropped people urls. Previously this logic was
+            # part of
+            # https://github.com/PostHog/posthog/blob/e8d7b2fe6047f5b31f704572cd3bebadddf50e0f/frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelStepTable.tsx#L483:L483
+            # NOTE: we default to 0 on order being None. On the frontend the typing
+            # suggests that order is always a number, so hopefully this will never
+            # actually be the case.
+            funnel_step = (step.order or 0) + 1
+            converted_people_filter = self._filter.with_data({"funnel_step": funnel_step})
+            dropped_people_filter = self._filter.with_data({"funnel_step": -funnel_step})
+
             if with_breakdown:
                 serialized_result.update({"breakdown": results[-1], "breakdown_value": results[-1]})
-                # important to not try and modify this value any how - as these are keys for fetching persons
+                # important to not try and modify this value any how - as these
+                # are keys for fetching persons
+
+                # Add in the breakdown to people urls as well
+                converted_people_filter = converted_people_filter.with_data({"funnel_step_breakdown": results[-1]})
+                dropped_people_filter = dropped_people_filter.with_data({"funnel_step_breakdown": results[-1]})
+
+            serialized_result.update(
+                {
+                    "converted_people_url": f"{self._base_uri}api/person/funnel/?{urllib.parse.urlencode(converted_people_filter.to_params())}",
+                    "dropped_people_url": (
+                        f"{self._base_uri}api/person/funnel/?{urllib.parse.urlencode(dropped_people_filter.to_params())}"
+                        # NOTE: If we are looking at the first step, there is no drop off,
+                        # everyone converted, otherwise they would not have been
+                        # included in the funnel. What should this mean for
+                        # unordered funnels? I'm not sure.
+                        if step.order
```

Comment:
use `index`, and make the check explicit please to make it clear that you're checking `step.index != 0` ?